Lesson7/deniz/style.css

```diff
@@ -0,0 +1,370 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+
+:root {
+    --white: #ffffff;
+    --black: #000000;
+    --grey-lighter: #555;
+    --black-darker: #242424;
+    --white-darker: #F2F2F2;
+    --green:  #9acd32;
+}
+
+
+main {
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: var(--white);
+    
+}
+
+.container-top {
+  width: 100%;
+}
+
+
+.h1-top {
+    padding-left: 2rem;
+  text-align: left;
+    color:  var(--black);
+    font-family: Arial, Helvetica, sans-serif;
```

Comment:
Unless you use a combination of different fonts, setting font-family once for body, :root or html selector is enough.